servers/v3-proxy-api/src/routes/validations/GetSchema.ts

```diff
@@ -108,6 +108,9 @@ export const V3GetSchema: Schema = {
         min: 0,
       },
     },
+    customSanitizer: {
+      options: (value) => (typeof value === 'string' ? parseInt(value) : value),
+    },
```

Comment:
This works but I think you can just use the builtin toInt

```suggestion
    toInt: true,
```